packages/app/src/cli/services/function/replay.ts

```diff
@@ -0,0 +1,27 @@
+import {joinPath} from '@shopify/cli-kit/node/path'
+import {readFile} from '@shopify/cli-kit/node/fs'
+import {readdirSync} from 'fs'
+
+export async function readFunctionRunsDirectory(functionPath: string) {
```

Comment:
Nit: Since this is a function that's exported, I think it'd be good to also type the return type so that the reader doesn't have to grok the implementation.

Reply:
reworked this so this is no longer exported 👍